src/sirius_interface.F

```diff
@@ -224,11 +238,152 @@ SUBROUTINE cp_sirius_create_env(pwdft_env)
       nkind = SIZE(atomic_kind_set)
       DO ikind = 1, nkind
          CALL get_atomic_kind(atomic_kind_set(ikind), &
-                              name=label, element_symbol=element_symbol, mass=mass, qeff=zeff)
-         NULLIFY (upf_pot)
-         CALL get_qs_kind(qs_kind_set(ikind), upf_potential=upf_pot)
-         CPASSERT(ASSOCIATED(upf_pot))
-         CALL sirius_add_atom_type(sctx, string(label), fname=string(upf_pot%filename), mass=REAL(mass, KIND=C_DOUBLE))
+                              name=label, element_symbol=element_symbol, mass=mass)
+         CALL get_qs_kind(qs_kind_set(ikind), zeff=zeff)
+         NULLIFY (upf_pot, gth_potential)
+         CALL get_qs_kind(qs_kind_set(ikind), upf_potential=upf_pot, gth_potential=gth_potential)
+         IF (ASSOCIATED(upf_pot)) THEN
+            CALL sirius_add_atom_type(sctx, string(label), fname=string(upf_pot%filename), &
+                                      mass=REAL(mass, KIND=C_DOUBLE))
+         ELSEIF (ASSOCIATED(gth_potential)) THEN
+            !
+            NULLIFY (atom_grid)
+            CALL allocate_grid_atom(atom_grid)
+            nmesh = 929
```

Comment:
accidentally hardcoded number of grid points?

Reply:
No. We didn't wanted to make this an input option, just use a hardcoded default.